readme.md

```diff
@@ -156,10 +156,10 @@ You can pass a query parameter `&hide=` to hide any specific stats with comma-se
 
 You can pass a query parameter `&show=` to show any specific additional stats with comma-separated values.
 
-> Options: `&show=reviews,discussions_started,discussions_answered`
+> Options: `&show=reviews,discussions_started,discussions_answered,prs_merged,merged_prs_percentage`
```

Comment:
Maybe for consistency, we can call it `prs_merged, prs_merged_percentage`. I am, however, also good with your current names if you think they are more apparent.

Reply:
I have update the code following your suggestion. Maybe it's better, but it hard for me to judge since i'm not english native speaker.